#VRML_SIM R2020a utf8
WorldInfo {
  info [
    "The user drives a Supervisor by the Keyboard which drives slaves robots by using an emitter device."
  ]
  title "Omniscience"
}
Viewpoint {
  orientation 0.9998777102302491 -0.014817904111507805 -0.004999430213136006 5.632314886642343
  position -0.2005385214730916 3.1155530170333257 3.3158101786493828
}
TexturedBackground {
}
TexturedBackgroundLight {
}
Robot {
  translation -0.8196595878973036 -0.00022713408974665145 -0.529704001084285
  rotation 0.00023053258872821771 0.9998987221155807 -0.014229981249265491 3.1097212393453404
  children [
    Emitter {
    }
    Compass {
      name "cp"
    }
    DEF MAIN_BODY Transform {
      translation 0 0.0415 0
      children [
        Shape {
          appearance PBRAppearance {
            baseColor 0.0820075 0.364731 0.8
            roughness 1
            metalness 0
          }
          geometry DEF BODY Cylinder {
            height 0.1
            radius 0.045
          }
        }
      ]
    }
    HingeJoint {
      jointParameters HingeJointParameters {
        position 3548.8428611679
        axis -1 0 0
        anchor 0 0.025 0
      }
      device [
        RotationalMotor {
          name "left wheel motor"
        }
      ]
      endPoint DEF LEFT_WHEEL Solid {
        translation -0.04499999999999993 0.025 0
        rotation -1 0 0 0.14167791861330162
        children [
          DEF WHEEL Transform {
            rotation 0 0 1 1.57
            children [
              Shape {
                appearance PBRAppearance {
                  baseColor 1 0 0
                  roughness 1
                  metalness 0
                }
                geometry Cylinder {
                  height 0.01
                  radius 0.025
                }
              }
            ]
          }
        ]
        name "left wheel"
        boundingObject DEF WHEEL Transform {
          rotation 0 0 1 1.57
          children [
            Shape {
              appearance PBRAppearance {
                baseColor 1 0 0
                roughness 1
                metalness 0
              }
              geometry Cylinder {
                height 0.01
                radius 0.025
              }
            }
          ]
        }
        physics DEF PHYSICS_WHEEL Physics {
          density -1
          mass 0.05
        }
        linearVelocity -2.0741245068935587e-17 7.452804329947653e-16 2.0075511287310946e-15
        angularVelocity 1.2398410060069793e-13 7.071032919851504e-14 -1.8863055778960278e-14
      }
    }
    HingeJoint {
      jointParameters HingeJointParameters {
        position 3680.5852257193224
        axis -1 0 0
        anchor 0 0.025 0
      }
      device [
        RotationalMotor {
          name "right wheel motor"
        }
      ]
      endPoint DEF RIGHT_WHEEL Solid {
        translation 0.04499999999999985 0.025 0
        rotation 0.9999999999999999 0 0 5.948714287915324
        children [
          DEF WHEEL Transform {
            rotation 0 0 1 1.57
            children [
              Shape {
                appearance PBRAppearance {
                  baseColor 1 0 0
                  roughness 1
                  metalness 0
                }
                geometry Cylinder {
                  height 0.01
                  radius 0.025
                }
              }
            ]
          }
        ]
        name "right wheel"
        boundingObject USE WHEEL
        physics DEF PHYSICS_WHEEL Physics {
          density -1
          mass 0.05
        }
        linearVelocity 2.076697168430849e-16 3.772221698618729e-17 7.862691993878061e-15
        angularVelocity 3.9230231339307413e-13 9.327118499068595e-14 -1.1385360228658836e-13
      }
    }
    DEF LEFT_EYE DistanceSensor {
      translation -0.02 0.063 -0.042
      rotation 0 1 0 2.07
      children [
        DEF INFRARED Transform {
          rotation 0 0 1 1.57
          children [
            Shape {
              appearance PBRAppearance {
                baseColor 0.975691 0.981481 0.0252992
                roughness 1
                metalness 0
              }
              geometry Cylinder {
                height 0.004
                radius 0.008
              }
            }
          ]
        }
      ]
      name "ds0"
      lookupTable [
        0 1024 0
        0.05 1024 0
        0.15 0 0
      ]
      numberOfRays 2
      aperture 1
    }
    DEF RIGHT_EYE DistanceSensor {
      translation 0.02 0.063 -0.042
      rotation 0 1 0 1.07
      children [
        DEF INFRARED Transform {
          rotation 0 0 1 1.57
          children [
            Shape {
              appearance PBRAppearance {
                baseColor 0.975691 0.981481 0.0252992
                roughness 1
                metalness 0
              }
              geometry Cylinder {
                height 0.004
                radius 0.008
              }
            }
          ]
        }
      ]
      name "ds1"
      lookupTable [
        0 1024 0
        0.05 1024 0
        0.15 0 0
      ]
      numberOfRays 2
      aperture 1
    }
    Receiver {
    }
    GPS {
      name "global"
    }
  ]
  name "robot1"
  boundingObject DEF MYBOT_BOUNDING Transform {
    translation 0 0.0415 0
    children [
      DEF BODY Cylinder {
        height 0.08
        radius 0.045
      }
    ]
  }
  physics DEF MYBOT_PHYSICS Physics {
    density -1
    mass 0.5
  }
  controller "arduino_2_webots_slave"
  linearVelocity -2.1791906969162067e-15 1.1937117960769684e-15 4.672261492531075e-15
  angularVelocity -3.142277164920175e-14 7.207751421259462e-14 8.61370210990393e-14
}
Robot {
  translation 0.8728004788530227 -0.00022712742434751604 0.5733372849088948
  rotation -0.9073170235205877 0.42040535562617226 0.005929231870393009 0.03136807605388057
  children [
    Emitter {
    }
    Compass {
      name "cp"
    }
    DEF MAIN_BODY Transform {
      translation 0 0.0415 0
      children [
        Shape {
          appearance PBRAppearance {
            baseColor 0.0820075 0.364731 0.8
            roughness 1
            metalness 0
          }
          geometry DEF BODY Cylinder {
            height 0.1
            radius 0.045
          }
        }
      ]
    }
    HingeJoint {
      jointParameters HingeJointParameters {
        position 3559.706844478918
        axis -1 0 0
        anchor 0 0.025 0
      }
      device [
        RotationalMotor {
          name "left wheel motor"
        }
      ]
      endPoint DEF LEFT_WHEEL Solid {
        translation -0.04499999999999993 0.025 0
        rotation 1 0 0 1.5607093847280504
        children [
          DEF WHEEL Transform {
            rotation 0 0 1 1.57
            children [
              Shape {
                appearance PBRAppearance {
                  baseColor 1 0 0
                  roughness 1
                  metalness 0
                }
                geometry Cylinder {
                  height 0.01
                  radius 0.025
                }
              }
            ]
          }
        ]
        name "left wheel"
        boundingObject DEF WHEEL Transform {
          rotation 0 0 1 1.57
          children [
            Shape {
              appearance PBRAppearance {
                baseColor 1 0 0
                roughness 1
                metalness 0
              }
              geometry Cylinder {
                height 0.01
                radius 0.025
              }
            }
          ]
        }
        physics DEF PHYSICS_WHEEL Physics {
          density -1
          mass 0.05
        }
        linearVelocity -1.800713594552051e-16 -2.1510606270592181e-16 1.6488276780738904e-15
        angularVelocity 9.151657821278214e-14 -1.3679768214853532e-13 -1.72734994053814e-13
      }
    }
    HingeJoint {
      jointParameters HingeJointParameters {
        position 3669.7212418893723
        axis -1 0 0
        anchor 0 0.025 0
      }
      device [
        RotationalMotor {
          name "right wheel motor"
        }
      ]
      endPoint DEF RIGHT_WHEEL Solid {
        translation 0.04499999999999986 0.025 0
        rotation 1 0 0 4.246327503506181
        children [
          DEF WHEEL Transform {
            rotation 0 0 1 1.57
            children [
              Shape {
                appearance PBRAppearance {
                  baseColor 1 0 0
                  roughness 1
                  metalness 0
                }
                geometry Cylinder {
                  height 0.01
                  radius 0.025
                }
              }
            ]
          }
        ]
        name "right wheel"
        boundingObject USE WHEEL
        physics DEF PHYSICS_WHEEL Physics {
          density -1
          mass 0.05
        }
        linearVelocity -2.0988146854532048e-15 5.5640166515114745e-16 1.4136962816417153e-14
        angularVelocity 6.695002336199684e-13 -1.1670484201749052e-13 -2.435502610744798e-13
      }
    }
    DEF LEFT_EYE DistanceSensor {
      translation -0.02 0.063 -0.042
      rotation 0 1 0 2.07
      children [
        DEF INFRARED Transform {
          rotation 0 0 1 1.57
          children [
            Shape {
              appearance PBRAppearance {
                baseColor 0.975691 0.981481 0.0252992
                roughness 1
                metalness 0
              }
              geometry Cylinder {
                height 0.004
                radius 0.008
              }
            }
          ]
        }
      ]
      name "ds0"
      lookupTable [
        0 1024 0
        0.05 1024 0
        0.15 0 0
      ]
      numberOfRays 2
      aperture 1
    }
    DEF RIGHT_EYE DistanceSensor {
      translation 0.02 0.063 -0.042
      rotation 0 1 0 1.07
      children [
        DEF INFRARED Transform {
          rotation 0 0 1 1.57
          children [
            Shape {
              appearance PBRAppearance {
                baseColor 0.975691 0.981481 0.0252992
                roughness 1
                metalness 0
              }
              geometry Cylinder {
                height 0.004
                radius 0.008
              }
            }
          ]
        }
      ]
      name "ds1"
      lookupTable [
        0 1024 0
        0.05 1024 0
        0.15 0 0
      ]
      numberOfRays 2
      aperture 1
    }
    Receiver {
    }
    GPS {
      name "global"
    }
  ]
  name "robot4"
  boundingObject DEF MYBOT_BOUNDING Transform {
    translation 0 0.0415 0
    children [
      DEF BODY Cylinder {
        height 0.08
        radius 0.045
      }
    ]
  }
  physics DEF MYBOT_PHYSICS Physics {
    density -1
    mass 0.5
  }
  controller "arduino_2_webots_slave"
  linearVelocity 1.0594976994703444e-15 -2.1600499167107045e-15 8.747041619287158e-15
  angularVelocity 4.856634819501902e-14 -1.380582912111915e-13 -7.797481743624377e-14
}
Robot {
  translation 1.041463489894591 -0.0002271329786638701 -0.8812188611205426
  rotation 0.8927840542241393 -0.4504422779509481 -0.006195704759728264 -0.03187850798715015
  children [
    Emitter {
    }
    Compass {
      name "cp"
    }
    DEF MAIN_BODY Transform {
      translation 0 0.0415 0
      children [
        Shape {
          appearance PBRAppearance {
            baseColor 0.0820075 0.364731 0.8
            roughness 1
            metalness 0
          }
          geometry DEF BODY Cylinder {
            height 0.1
            radius 0.045
          }
        }
      ]
    }
    HingeJoint {
      jointParameters HingeJointParameters {
        position 3570.074853630125
        axis -1 0 0
        anchor 0 0.025 0
      }
      device [
        RotationalMotor {
          name "left wheel motor"
        }
      ]
      endPoint DEF LEFT_WHEEL Solid {
        translation -0.044999999999999936 0.025 0
        rotation 1 0 0 3.75907084788029
        children [
          DEF WHEEL Transform {
            rotation 0 0 1 1.57
            children [
              Shape {
                appearance PBRAppearance {
                  baseColor 1 0 0
                  roughness 1
                  metalness 0
                }
                geometry Cylinder {
                  height 0.01
                  radius 0.025
                }
              }
            ]
          }
        ]
        name "left wheel"
        boundingObject DEF WHEEL Transform {
          rotation 0 0 1 1.57
          children [
            Shape {
              appearance PBRAppearance {
                baseColor 1 0 0
                roughness 1
                metalness 0
              }
              geometry Cylinder {
                height 0.01
                radius 0.025
              }
            }
          ]
        }
        physics DEF PHYSICS_WHEEL Physics {
          density -1
          mass 0.05
        }
        linearVelocity 1.8651523246317387e-15 -1.2780096449419104e-15 1.2801564671848371e-14
        angularVelocity 6.271828882718801e-13 9.538473819488986e-14 -5.643795511974427e-14
      }
    }
    HingeJoint {
      jointParameters HingeJointParameters {
        position 3683.161233024145
        axis -1 0 0
        anchor 0 0.025 0
      }
      device [
        RotationalMotor {
          name "right wheel motor"
        }
      ]
      endPoint DEF RIGHT_WHEEL Solid {
        translation 0.044999999999999866 0.025 0
        rotation 1 0 0 3.3727069830925998
        children [
          DEF WHEEL Transform {
            rotation 0 0 1 1.57
            children [
              Shape {
                appearance PBRAppearance {
                  baseColor 1 0 0
                  roughness 1
                  metalness 0
                }
                geometry Cylinder {
                  height 0.01
                  radius 0.025
                }
              }
            ]
          }
        ]
        name "right wheel"
        boundingObject USE WHEEL
        physics DEF PHYSICS_WHEEL Physics {
          density -1
          mass 0.05
        }
        linearVelocity 7.132857471330408e-15 9.819799375177827e-16 3.91225317097511e-15
        angularVelocity 2.498754940118265e-13 1.338053452529968e-13 -2.53169424339204e-13
      }
    }
    DEF LEFT_EYE DistanceSensor {
      translation -0.02 0.063 -0.042
      rotation 0 1 0 2.07
      children [
        DEF INFRARED Transform {
          rotation 0 0 1 1.57
          children [
            Shape {
              appearance PBRAppearance {
                baseColor 0.975691 0.981481 0.0252992
                roughness 1
                metalness 0
              }
              geometry Cylinder {
                height 0.004
                radius 0.008
              }
            }
          ]
        }
      ]
      name "ds0"
      lookupTable [
        0 1024 0
        0.05 1024 0
        0.15 0 0
      ]
      numberOfRays 2
      aperture 1
    }
    DEF RIGHT_EYE DistanceSensor {
      translation 0.02 0.063 -0.042
      rotation 0 1 0 1.07
      children [
        DEF INFRARED Transform {
          rotation 0 0 1 1.57
          children [
            Shape {
              appearance PBRAppearance {
                baseColor 0.975691 0.981481 0.0252992
                roughness 1
                metalness 0
              }
              geometry Cylinder {
                height 0.004
                radius 0.008
              }
            }
          ]
        }
      ]
      name "ds1"
      lookupTable [
        0 1024 0
        0.05 1024 0
        0.15 0 0
      ]
      numberOfRays 2
      aperture 1
    }
    Receiver {
    }
    GPS {
      name "global"
    }
  ]
  name "robot5"
  boundingObject DEF MYBOT_BOUNDING Transform {
    translation 0 0.0415 0
    children [
      DEF BODY Cylinder {
        height 0.08
        radius 0.045
      }
    ]
  }
  physics DEF MYBOT_PHYSICS Physics {
    density -1
    mass 0.5
  }
  controller "arduino_2_webots_slave"
  linearVelocity 2.4444522155904656e-15 3.808509063674137e-15 6.8636394457660326e-15
  angularVelocity -8.113872702566857e-14 1.1861049132514388e-13 5.3441286887444366e-14
}
Robot {
  translation 0.13706966255331182 -0.00022712675768192525 0.4626516184238295
  rotation 0.6509682300129798 0.7590284601787756 0.010777761938595021 -0.04371737169518259
  children [
    Emitter {
    }
    Compass {
      name "cp"
    }
    GPS {
      name "global"
    }
    DEF MAIN_BODY Transform {
      translation 0 0.0415 0
      children [
        Shape {
          appearance PBRAppearance {
            baseColor 0.0820075 0.364731 0.8
            roughness 1
            metalness 0
          }
          geometry DEF BODY Cylinder {
            height 0.1
            radius 0.045
          }
        }
      ]
    }
    HingeJoint {
      jointParameters HingeJointParameters {
        position 1062.135308293986
        axis -1 0 0
        anchor 0 0.025 0
      }
      device [
        RotationalMotor {
          name "left wheel motor"
        }
      ]
      endPoint DEF LEFT_WHEEL Solid {
        translation -0.04499999999999985 0.025 0
        rotation -1 0 0 1.5755066878155775
        children [
          DEF WHEEL Transform {
            rotation 0 0 1 1.57
            children [
              Shape {
                appearance PBRAppearance {
                  baseColor 1 0 0
                  roughness 1
                  metalness 0
                }
                geometry Cylinder {
                  height 0.01
                  radius 0.025
                }
              }
            ]
          }
        ]
        name "left wheel"
        boundingObject DEF WHEEL Transform {
          rotation 0 0 1 1.57
          children [
            Shape {
              appearance PBRAppearance {
                baseColor 1 0 0
                roughness 1
                metalness 0
              }
              geometry Cylinder {
                height 0.01
                radius 0.025
              }
            }
          ]
        }
        physics DEF PHYSICS_WHEEL Physics {
          density -1
          mass 0.05
        }
        linearVelocity 3.637848450324165e-15 -2.133507031838845e-16 -1.3654952303598475e-16
        angularVelocity -1.1559858963449143e-14 8.969779972778902e-14 -6.125426689973779e-14
      }
    }
    HingeJoint {
      jointParameters HingeJointParameters {
        position 1124.2967355311876
        axis -1 0 0
        anchor 0 0.025 0
      }
      device [
        RotationalMotor {
          name "right wheel motor"
        }
      ]
      endPoint DEF RIGHT_WHEEL Solid {
        translation 0.044999999999999964 0.025 0
        rotation -1 0 0 1.3024008532211606
        children [
          DEF WHEEL Transform {
            rotation 0 0 1 1.57
            children [
              Shape {
                appearance PBRAppearance {
                  baseColor 1 0 0
                  roughness 1
                  metalness 0
                }
                geometry Cylinder {
                  height 0.01
                  radius 0.025
                }
              }
            ]
          }
        ]
        name "right wheel"
        boundingObject USE WHEEL
        physics DEF PHYSICS_WHEEL Physics {
          density -1
          mass 0.05
        }
        linearVelocity 2.0962439401827203e-15 -7.102637020271591e-17 -8.193912285144774e-15
        angularVelocity -4.0594425818932057e-13 6.943400507154261e-14 2.821528266466601e-14
      }
    }
    DEF LEFT_EYE DistanceSensor {
      translation -0.02 0.063 -0.042
      rotation 0 1 0 2.07
      children [
        DEF INFRARED Transform {
          rotation 0 0 1 1.57
          children [
            Shape {
              appearance PBRAppearance {
                baseColor 0.975691 0.981481 0.0252992
                roughness 1
                metalness 0
              }
              geometry Cylinder {
                height 0.004
                radius 0.008
              }
            }
          ]
        }
      ]
      name "ds0"
      lookupTable [
        0 1024 0
        0.05 1024 0
        0.15 0 0
      ]
      numberOfRays 2
      aperture 1
    }
    DEF RIGHT_EYE DistanceSensor {
      translation 0.02 0.063 -0.042
      rotation 0 1 0 1.07
      children [
        DEF INFRARED Transform {
          rotation 0 0 1 1.57
          children [
            Shape {
              appearance PBRAppearance {
                baseColor 0.975691 0.981481 0.0252992
                roughness 1
                metalness 0
              }
              geometry Cylinder {
                height 0.004
                radius 0.008
              }
            }
          ]
        }
      ]
      name "ds1"
      lookupTable [
        0 1024 0
        0.05 1024 0
        0.15 0 0
      ]
      numberOfRays 2
      aperture 1
    }
    Receiver {
    }
  ]
  name "robot2"
  boundingObject DEF MYBOT_BOUNDING Transform {
    translation 0 0.0415 0
    children [
      DEF BODY Cylinder {
        height 0.08
        radius 0.045
      }
    ]
  }
  physics DEF MYBOT_PHYSICS Physics {
    density -1
    mass 0.5
  }
  controller "arduino_2_webots_slave"
  linearVelocity 4.657426594345414e-15 2.842170943040401e-16 -4.3131146563203926e-15
  angularVelocity -3.2549770026521658e-15 8.954811048829434e-14 -1.0197259515602139e-13
}
Robot {
  translation -1.0596045108887047 -0.00022712680831592225 0.4803671986909981
  rotation 0.01384439998652372 0.9998029205005994 0.014228589089771941 -1.5983872790985894
  children [
    Emitter {
    }
    Compass {
      name "cp"
    }
    GPS {
      name "global"
    }
    DEF MAIN_BODY Transform {
      translation 0 0.0415 0
      children [
        Shape {
          appearance PBRAppearance {
            baseColor 0.0820075 0.364731 0.8
            roughness 1
            metalness 0
          }
          geometry DEF BODY Cylinder {
            height 0.1
            radius 0.045
          }
        }
      ]
    }
    HingeJoint {
      jointParameters HingeJointParameters {
        position 3531.1942861385996
        axis -1 0 0
        anchor 0 0.025 0
      }
      device [
        RotationalMotor {
          name "left wheel motor"
        }
      ]
      endPoint DEF LEFT_WHEEL Solid {
        translation -0.044999999999999825 0.025 0
        rotation 1 0 0 4.9405264963282
        children [
          DEF WHEEL Transform {
            rotation 0 0 1 1.57
            children [
              Shape {
                appearance PBRAppearance {
                  baseColor 1 0 0
                  roughness 1
                  metalness 0
                }
                geometry Cylinder {
                  height 0.01
                  radius 0.025
                }
              }
            ]
          }
        ]
        name "left wheel"
        boundingObject DEF WHEEL Transform {
          rotation 0 0 1 1.57
          children [
            Shape {
              appearance PBRAppearance {
                baseColor 1 0 0
                roughness 1
                metalness 0
              }
              geometry Cylinder {
                height 0.01
                radius 0.025
              }
            }
          ]
        }
        physics DEF PHYSICS_WHEEL Physics {
          density -1
          mass 0.05
        }
        linearVelocity -9.069569523214729e-11 -1.067929269711375e-12 2.586081815224286e-09
        angularVelocity 9.173346302517238e-12 -2.383224825781718e-10 6.642794857607764e-10
      }
    }
    HingeJoint {
      jointParameters HingeJointParameters {
        position 3631.7237517610124
        axis -1 0 0
        anchor 0 0.025 0
      }
      device [
        RotationalMotor {
          name "right wheel motor"
        }
      ]
      endPoint DEF RIGHT_WHEEL Solid {
        translation 0.0449999999999997 0.025 0
        rotation 1 0 0 4.544705788788539
        children [
          DEF WHEEL Transform {
            rotation 0 0 1 1.57
            children [
              Shape {
                appearance PBRAppearance {
                  baseColor 1 0 0
                  roughness 1
                  metalness 0
                }
                geometry Cylinder {
                  height 0.01
                  radius 0.025
                }
              }
            ]
          }
        ]
        name "right wheel"
        boundingObject USE WHEEL
        physics DEF PHYSICS_WHEEL Physics {
          density -1
          mass 0.05
        }
        linearVelocity 6.191204220502539e-11 -9.211760175399039e-12 -2.5928473752906276e-09
        angularVelocity 1.390727643071275e-10 -2.3741797371137365e-10 6.678899643879032e-10
      }
    }
    DEF LEFT_EYE DistanceSensor {
      translation -0.02 0.063 -0.042
      rotation 0 1 0 2.07
      children [
        DEF INFRARED Transform {
          rotation 0 0 1 1.57
          children [
            Shape {
              appearance PBRAppearance {
                baseColor 0.975691 0.981481 0.0252992
                roughness 1
                metalness 0
              }
              geometry Cylinder {
                height 0.004
                radius 0.008
              }
            }
          ]
        }
      ]
      name "ds0"
      lookupTable [
        0 1024 0
        0.05 1024 0
        0.15 0 0
      ]
      numberOfRays 2
      aperture 1
    }
    DEF RIGHT_EYE DistanceSensor {
      translation 0.02 0.063 -0.042
      rotation 0 1 0 1.07
      children [
        DEF INFRARED Transform {
          rotation 0 0 1 1.57
          children [
            Shape {
              appearance PBRAppearance {
                baseColor 0.975691 0.981481 0.0252992
                roughness 1
                metalness 0
              }
              geometry Cylinder {
                height 0.004
                radius 0.008
              }
            }
          ]
        }
      ]
      name "ds1"
      lookupTable [
        0 1024 0
        0.05 1024 0
        0.15 0 0
      ]
      numberOfRays 2
      aperture 1
    }
    Receiver {
    }
  ]
  name "robot3"
  boundingObject DEF MYBOT_BOUNDING Transform {
    translation 0 0.0415 0
    children [
      DEF BODY Cylinder {
        height 0.08
        radius 0.045
      }
    ]
  }
  physics DEF MYBOT_PHYSICS Physics {
    density -1
    mass 0.5
  }
  controller "arduino_2_webots_slave"
  linearVelocity -3.6242112006439194e-11 -2.0510810827545355e-12 1.353736828885177e-11
  angularVelocity 7.380625582466859e-11 -2.385798593767739e-10 6.662475697903835e-10
}
DEF Supervisor Robot {
  children [
    Emitter {
    }
  ]
  name "Driver"
  controller "arduino_2_webots_driver"
  supervisor TRUE
}
DEF _ RectangleArena {
  translation -0.0447275 0 0.111226
  floorSize 3 2.3
  floorTileSize 0.2 0.2
}
